packages/minify/README.md

```diff
@@ -1,8 +1,11 @@
 # Neutrino Minify Middleware
 
-`@neutrinojs/minify` is Neutrino middleware for minifying source code using
-[`BabelMinifyWebpackPlugin`](https://www.npmjs.com/package/babel-minify-webpack-plugin). This middleware is usually only
-added during production builds.
+`@neutrinojs/minify` is Neutrino middleware for minifying assets. It is an aggregate of the following middlewares:
+
+- [@neutrinojs/babel-minify](../../packages/babel-minify/README.md)
```

Comment:
For the `packages` directory these should be absolute URLs to `https://neutrino.js.org`. They are relative when in `docs/packages`.